style-guide.rst

```diff
@@ -2,15 +2,14 @@
 
 .. _style-guide:
 
-============================
 reStructuredText style guide
 ============================
 
-The documentation files use `reStructuredText (rST) <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_ syntax. |x wokeignore:rule=master x|
+The documentation files use `reStructuredText`_ (rST) syntax.
```

Comment:
Minor: I've seen RST, ReST, and reST - but never rST.

Reply:
I had even double-checked to make sure - but I must have expected to see "rST" so much that I didn't notice it was actually "reST". :laughing: